### DISCUSSION TITLE
How to get a list of all the channels in a .mf4 file?

### DISCUSSION THREAD

**Discussion by ronitch** [danielhrisca/asammdf#729] [Q&A]
I want to list the all the channel names in a .mf4 file, [here](https://asammdf.readthedocs.io/en/2.1.2/mdf4.html) it says that the _channels_ argument in the _groups_ attribute does that but I am unable to figure out how to write this code.

**Comment by danielhrisca** [comment]
Hello , you need to check the ``channels_db`` attribute
